README.md

```diff
@@ -59,7 +59,7 @@ The tenants database connection details are configured [via environment variable
 Instructions to build a WAR file
 ============
 1. Clone the repository or download and extract the archive file to your local directory.
-2. Run `./gradlew clean war` to build a traditional WAR file which will be created at `build/libs` directory.
+2. Run `./gradlew clean war` to build a traditional WAR file which will be created at `fineract-provider/build/libs` directory.
 3. Deploy this WAR to your Tomcat v9 Servlet Container.
 
 We recommend using the JAR instead of the WAR file deployment, because it's much easier.
```

Comment:
@vorburger @vidakovic 

Is this line necessary?
Is it easier to configure SSL certificate for JAR file instead of  Tomcat WAR?